test/box/update.result

```diff
@@ -1821,14 +1813,8 @@ t:update({{'=', '[4]', {1}}, {'+', '[4][1]', 1}})
 - [1, {}, [], [2]]
 ...
 -- Same about updating internals of a newly created bar field.
-t:update({{'=', '[4][6]', {a = 33}}, {'+', '[4][6].aa', 1}})
----
-- error: 'Field ''[4][6].aa'' UPDATE error: double update of the same field'
-...
-t:update({{'=', '[4][4]', {3}}, {'+', '[4][4][1]', 1}})
----
-- error: 'Field ''[4][4][1]'' UPDATE error: double update of the same field'
-...
+-- t:update({{'=', '[4][6]', {a = 33}}, {'+', '[4][6].aa', 1}})
+-- t:update({{'=', '[4][4]', {3}}, {'+', '[4][4][1]', 1}})
```

Comment:
Ditto.

Reply:
Moved the below hunk to a later patch so now I don't need commenting these tests.
```diff
-               if (old_token.type == JSON_TOKEN_END) {
-                       /*
-                        * The bar path ended. It means the new operation either
-                        * is trying to update exactly the same field, or go
-                        * even deeper.
-                        */
-                       xrow_update_err_double(new_op);
-                       return NULL;
-               }
```
In the previous version of the patch set I deleted hunk too early -  it works until the patch where it deleted now.